v19.2/verify-constraint-conformance.md

```diff
@@ -0,0 +1,565 @@
+- [`system.replication_critical_localities`](#system-replication_critical_localities) shows which localities in your cluster have data that is at risk in the event of node failures.
+- [`system.reports_meta`](#system-reports_meta) lists the IDs and times when the reports ran that generated the data in the `system.replication_*` tables.
+- [`crdb_internal.zones`](#crdb_internal-zones) can be used with the tables above to figure out the database and table names where the non-conforming or at-risk data is located.
+
+To configure how often the conformance reports run, adjust the XXX cluster setting.
+
+Only members of the `admin` role can access these tables. By default, the `root` user belongs to the `admin` role. For more information about users and roles, see [Manage Users](authorization.html#create-and-manage-users) and [Manage Roles](roles.html).
+
+{{site.data.alerts.callout_info}}
+The `system.replication_*` tables listed above are only populated in clusters where the zone configurations have been changed from the defaults.
+{{site.data.alerts.end}}
+
+### system.replication_stats
+
+The `system.replication_stats` table shows information about whether data is under-replicated, over-replicated, or unavailable.
+
+For an example using this table, see [Find out which databases and tables have under-replicated ranges](#find-out-which-databases-and-tables-have-under-replicated-ranges).
+
+{% include copy-clipboard.html %}
+~~~ sql
+SHOW COLUMNS FROM system.replication_stats;
+~~~
+
+| Column name             | Data type          | Description                                                                            |
+|-------------------------+--------------------+----------------------------------------------------------------------------------------|
+| zone_id                 | [`INT8`](int.html) | The ID of the [replication zone](configure-zone.html).                                 |
+| subzone_id              | [`INT8`](int.html) | The ID of the subzone (i.e., [partition](partition-by.html)).                          |
+| report_id               | [`INT8`](int.html) | The ID of the [report](#system-reports_meta) that generated this data.                 |
+| total_ranges            | [`INT8`](int.html) | Total [ranges](architecture/overview.html#architecture-range) in the cluster.          |
+| unavailable_ranges      | [`INT8`](int.html) | Unavailable ranges.                                                                    |
+| under_replicated_ranges | [`INT8`](int.html) | [Under-replicated](admin-ui-replication-dashboard.html#under-replicated-ranges) ranges. |
+| over_replicated_ranges  | [`INT8`](int.html) | Over-replicated ranges.                                                                |
+
+### system.replication_critical_localities
+
+The `system.replication_critical_localities` table shows which localities in your cluster have data that is at risk in the event of node failures in your cluster.
```

Comment:
Like I was saying above, I'd explain what a critical locality is more and explain that it's not necessarily bad to have them.
I'd also find a way to work the term "locality tier" into this discussion.

Reply:
Where can I learn more about what is a "critical locality"? (other than your previous comment)

grepping the code & RFCs for "critical locality" returned one result from your PR adding these reports :-) 

Despite the existence of [Configure Replication Zones](https://www.cockroachlabs.com/docs/stable/configure-replication-zones.html) and friends, according to the Googles we have not really documented those terms in a public-facing way yet. :-/

(I did find lots of "locality tier" comments in the code so I'll read them and learn something about that in the meantime.)